Description: Fix rados-classes search path.
 .
 The commit 3bee4b0 changed the
 base of the osd_class_dir path from CMAKE_INSTALL_FULL_LIBDIR to
 CMAKE_INSTALL_LIBDIR. This change aligns the default value with the
 correct value.
Author: Chris MacNaughton <chris.macnaughton@ubuntu.com>
Forwarded: https://github.com/ceph/ceph/pull/47676
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/ceph/+bug/1986747

--- a/src/common/options/osd.yaml.in
+++ b/src/common/options/osd.yaml.in
@@ -534,7 +534,7 @@
 - name: osd_class_dir
   type: str
   level: advanced
-  default: @CMAKE_INSTALL_LIBDIR@/rados-classes
+  default: @CMAKE_INSTALL_FULL_LIBDIR@/rados-classes
   fmt_desc: The class path for RADOS class plug-ins.
   with_legacy: true
 - name: osd_open_classes_on_start
